codegen/cuekind/def.cue

```diff
@@ -70,6 +70,10 @@ Schema: {
 	_specIsNonEmpty: spec & struct.MinFields(0)
 }
 
+#AdmissionCapability: {
+	operations: [...string]
```

Comment:
I think this is is a good idea, but I'll implement this in-code rather than in-CUE, because I just tried it in-CUE, and the error that it returns if an invalid value is passed (used `["create","update","invalid"]`) is not very comprehensible:
```
Error: issue.apiResource.validation: 7 errors in empty disjunction: (and 7 more errors)
```

Reply:
Errors from the jenny validation instead:
```
* ManifestGenerator: validation operations error: invalid operation "invalid"
```
(for `*` present alongside other operations):
```
* ManifestGenerator: validation operations error: cannot use any ('*') operation alongside named operations
```

I've also updated `app.ValidationCapability.Operations` and `app.MutationCapability.Operations` to be of `[]app.AdmissionOperation` type instead of `[]string`, but I'm of mixed feelings that this the right change. Thoughts there?

Reply:
SGTM, thank you!